### ISSUE TITLE
Undefined symbols for architecture i386 when running test

### ISSUE THREAD

**Issue by giuliomoro** [pingdynasty/OwlProgram#54]
```
$ make TOOLROOT=/opt/local/bin/ test -j6 PATCHNAME=ShortArrayTest

Undefined symbols for architecture i386:
  "Patch::getSampleRate()", referenced from:
      AdsrEnvelope::setTimeBase(unsigned int) in Envelope.o
      WavetableOscillator::setTimeBase(unsigned int) in WavetableOscillator.o
      Oscillator::setPeriod(float) in WavetableOscillator.o
      Oscillator::setPeriod(float) in PolyBlepOscillator.o
```

This on the timebase branch

in `/opt/local/bin/` I have an actual gcc 4.9 installed through macports.

Running the same command from the shortmath branch works fine.

**Comment by giuliomoro** [comment]
Closed a8971e7

**Comment by pingdynasty** [comment]
fixed
